wikimedia/wikipedia-ios
# Native Editor - Add strikethrough support

**Phabricator:** 
https://phabricator.wikimedia.org/T348072 (strikethrough portion)

### Notes
Implements syntax highlighting and actions for strikethrough.

### Test Steps
1. Open article from Staging scheme.
2. Type `<s>Test</s>`. Confirm tags are syntax highlighted green.
3. Place cursor within `<s></s>` tags
4. Confirm strikethrough button shows selected state.
5. Tap strikethrough button.
6. Confirm selection range expands to contained text and surrounding `s` tags are removed. Confirm strikethrough button shows unselected state.
7. Tap strikethrough button again.
8. Confirm surrounding `s` tags are added to selection.
